core/services/vrf/integration_v2_test.go

```diff
@@ -372,6 +376,57 @@ func createVRFJobs(
 	return
 }
 
+func requestRandomnessForWrapper(
+	t *testing.T,
+	vrfWrapperConsumer vrfv2_wrapper_consumer_example.VRFV2WrapperConsumerExample,
+	consumerOwner *bind.TransactOpts,
+	keyHash common.Hash,
+	subID uint64,
+	numWords uint32,
+	cbGasLimit uint32,
+	uni coordinatorV2Universe,
+	wrapperOverhead uint32,
+) (*big.Int, uint64) {
+	minRequestConfirmations := uint16(3)
+	fmt.Println(consumerOwner)
+	fmt.Println(cbGasLimit)
+	fmt.Println(numWords)
```

Comment:
Small nit: prefer `t.Log()` in tests, it'll log things in a more obvious way.

Reply:
Those shouldn't even be there tbh, good catch!